src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -96,6 +95,21 @@ public function __construct(BasketManagerInterface $basketManager, BasketElement
      */
     public function getBasketsAction(ParamFetcherInterface $paramFetcher)
     {
+        $orderByQueryParam = new QueryParam();
+        $orderByQueryParam->name = 'orderBy';
+        $orderByQueryParam->requirements = 'ASC|DESC';
+        $orderByQueryParam->nullable = true;
+        $orderByQueryParam->strict = true;
+        $orderByQueryParam->description = 'Query baskets basket by clause (key is field, value is direction)';
```

Comment:
Please take this as an occasion to change "Query baskets basket by" to "Basket query order by", this looks silly.

Reply:
Maybe "Basket query order**ed** by"?

Reply:
Or maybe like "Sort results by (key is field, value is direction)"

Reply:
> Sort results by (key is field, value is direction)

Feels less SQL-ish, might be better indeed 🤔

Reply:
Maybe just "sort specification (key is field, value is direction)" ?

Reply:
"sort specification for the current resultset (key is field, value is direction)"?

Reply:
Maybe "sort specification for the ~current~ resultset (key is field, value is direction)"

> Sort specification for the resultset (key is field, value is direction)